### PR TITLE
contrib/coturn: new package (4.6.2)

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

File: contrib/coturn/template.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+make_cmd = "gmake"
+make_dir = "."
+make_check_target = "test"
+hostmakedepends = ["gmake", "pkgconf"]
+makedepends = [
+    "openssl-devel",
+    "libevent-devel",
+    "linux-headers",
+    "sqlite-devel",
+    "hiredis-devel",
```
Comment (nekopsykose) [inline contrib/coturn/template.py:16]: sort

Comment (ttyyls) [reply contrib/coturn/template.py]: sorted

File: contrib/coturn/update.py [hunk 2]
```diff
@@ -0,0 +1 @@
+ignore = True
```
Comment (nekopsykose) [inline contrib/coturn/update.py:1]: use api.github.com

Comment (nekopsykose) [reply contrib/coturn/update.py:1]: see contrib/rust-analyzer/update.py etc

Comment (ttyyls) [reply contrib/coturn/update.py]: thanks

**Pull request merged**